code/datums/movement_controller/pod.dm

```diff
@@ -138,7 +138,8 @@
 
 				vel_max /= (owner.speed ? owner.speed : 1)
 
-				if (velocity_magnitude > vel_max)
+				// lower maximum velocities result in pod being very slow to turn, so a base max velocity is set
+				if (velocity_magnitude > vel_max * max(src.owner.speedmod * src.owner.engine.speedmod, 1))
 					velocity_x /= velocity_magnitude
```

Comment:
Implementation feels unintuitive - pods with a slow engine will have an unchanged top speed, just slower (de)acceleration, but a 'positive' speedmod will improve acceleration *and* affect top speed.

Reply:
Okay I'll remove the deceleration part. Was having issues with pod turn speed being unacceptably slow for low max velocity. I find the code hard to understand, may be a better way I'm not seeing